src/group/organizationGroup/organizationGroup.controller.ts

```diff
@@ -156,63 +266,77 @@ export class OrganizationGroup {
   private static async disownChild(parentID: string, childID: string): Promise<IOrganizationGroup> {
     if (!parentID) return;
     const parent = await OrganizationGroup.getOrganizationGroupOld(parentID);
-    _.pull(<string[]>parent.children, childID);
+    _.remove(<string[]>parent.children, (item) => { return item.toString() === childID; });
     if (parent.children.length === 0) {
       parent.isALeaf = true;
     } else parent.isALeaf = false;
     return await OrganizationGroup.updateOrganizationGroup(parent.id, parent);
   }
 
-  private static async getAncestors(organizationGroupID: string): Promise<string[]> {
+  /**
+   * Return array of ids ancestors of group
+   * @param organizationGroupID ID of group we want its ancestors
+   */
+  private static async getIDAncestors(organizationGroupID: string): Promise<string[]> {
     const organizationGroup = await OrganizationGroup.getOrganizationGroupOld(organizationGroupID);
     if (!organizationGroup.ancestors) return [];
     return <string[]>organizationGroup.ancestors;
   }
 
+  /**
+   * Get ancestors of group 
+   * @param organizationGroupID ID of group
+   * @param cond Condition of query
+   */
+  private static async getAncestors(organizationGroupID: string, cond?: Object): Promise<IOrganizationGroup[]> {
+    const organizationGroup = await OrganizationGroup.getOrganizationGroupOld(organizationGroupID);
+    if (!organizationGroup.ancestors) return [];
+
+    // If there are conditions adding them to the request, otherwise no
+    const ancestorObjects: IOrganizationGroup[] = cond ?
```

Comment:
can just send "cond", and make it equal to "{}" by default

Reply:
Done